nix/devcontainer/root/etc/shadow

```diff
@@ -0,0 +1,32 @@
+root:!::0:::::
```

Comment:
i honestly haven't looked at this in quite a while but i am somewhat surprised we have to create all of these files manually - at the very least for the shadow file i seem to remember that there is some convenience script thing that can be invoked to create it - Probably used in the docker related tests in nixpkgs somewhere.

Reply:
I stole this from https://github.com/nix-community/docker-nixpkgs/tree/master/images/devcontainer. To be clear, I have no idea what I'm doing, so this might be completely useless :sweat_smile:

Reply:
This produces a nonfunctional image for some reason :(